nservicebus/upgrades/3to4.md

```diff
@@ -146,6 +146,7 @@ NServiceBus PowerShell cmdlets have moved to NServiceBus.PowerShell.dll. See [Ma
 
 `configure.JsonSerializer()` has been obsoleted, please use `Configure.Serialization.Json();` instead. More info on supported serializers [here](/nservicebus/serialization/).
 
+The `XmlSerializer` will now automatically escape outgoing messages containing invalid characters. Version 3 endpoints will not be able to process those messages correctly.
```

Comment:
@simoncropp we need some help from the doco god here :)

On Thu, Oct 8, 2015 at 12:34 PM, Igal Tabachnik <notifications@github.com>
wrote:

> In nservicebus/upgrades/3to4.md
> <https://github.com/Particular/docs.particular.net/pull/867#discussion_r41498473>
> :
>
> > @@ -146,6 +146,7 @@ NServiceBus PowerShell cmdlets have moved to NServiceBus.PowerShell.dll. See [Ma
> >
> >  `configure.JsonSerializer()` has been obsoleted, please use `Configure.Serialization.Json();` instead. More info on supported serializers [here](/nservicebus/serialization/).
> >
> > +The `XmlSerializer` will now automatically escape outgoing messages containing invalid characters. Version 3 endpoints will not be able to process those messages correctly.
>
> That's what I had initially, but it read like tautology...
> Because it sounds odd: v4 endpoints can handle invalid characters, but v3
> cannot, so I wanted to emphasize that v3 is at fault here, not v4. Which is
> why you should upgrade :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/867/files#r41498473>
> .
>